core/domain/user_jobs_one_off_test.py

```diff
@@ -1218,7 +1218,12 @@ def test_that_last_created_time_is_updated(self):
 
         user_settings = user_services.get_user_settings(self.owner_id)
         user_settings.last_created_an_exploration = None
-        user_services._save_user_settings(user_settings)  # pylint: disable=protected-access
+        user_models.UserSettingsModel(
```

Comment:
You don't really need some of the previous lines if you are taking this approach, right? (I mean, you're creating a domain object and not exactly using it...)

Ditto elsewhere.

Reply:
@seanlip I will remove the lines.